### DISCUSSION TITLE
Fill in the grid ✏️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1280] [Galaxy Trivia] [answered]
Using the letters `AAEEHHLLLLMMNNOOSSTT` make the grid read the same across ➡️ as down ⬇️.

| C |  |  |  |  |
|--------|--------|--------|--------|--------|
| | E | | | |
| | | I | | |
| | | | P | |
| | | | | H |

**Comment by tomerf** [comment, marked as answer]
| C | H | A | S | M |
|---|---|---|---|---|
| H | E | L | L | O |
| A | L | I | E | N |
| S | L | E | P | T |
| M | O | N | T | H |
